docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,20 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages when building on Read The Docs
+def setup(app: Sphinx) -> dict[str, bool | str]:
+
+    if 'redirects' in app.tags:
+
+        redirects_config_path = DOCS_ROOT_DIR.parent / "declarative-configs" / "ansible_redirects.toml"
+        redirects = tomllib.loads(redirects_config_path.read_text())
+
+        app.setup_extension('sphinx_reredirects') # redirect pages that have been restructured or removed
+        app.config.redirects = redirects
```

Comment:
Let's set the config before loading the extension to be on the safe side:
```suggestion
        app.config.redirects = redirects
        app.setup_extension('sphinx_reredirects') # redirect pages that have been restructured or removed
```